### PR TITLE
Add Azure.Core changelogs

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: releases/2020-06/dotnet.md [hunk 1]
```diff
@@ -40,6 +43,30 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
 - Client library now targets the service's v3.0 API, instead of the v3.0-preview.1 API.
 - Updated the models to correspond with service changes.
 
+### Core Experimental [Changelog](https://github.com/Azure/azure-sdk-for-net/blob/Azure.Core.Experimental_0.1.0-preview.1/sdk/core/Azure.Core.Experimental/CHANGELOG.md)
```
Comment (ShivangiReja) [inline releases/2020-06/dotnet.md:46]: Changelog link should point to the particular date. In your case it should be - https://github.com/Azure/azure-sdk-for-net/blob/Azure.Core.Experimental_0.1.0-preview.1/sdk/core/Azure.Core.Experimental/CHANGELOG.md#010-preview1-2020-06-04

Comment (ShivangiReja) [reply releases/2020-06/dotnet.md:46]: Please update the other links as well.

Comment (pakrym) [reply releases/2020-06/dotnet.md:46]: Do we now point to the changelog in master instead of tag?

Comment (ShivangiReja) [reply releases/2020-06/dotnet.md:46]: Ah, I missed that you are using tag, not master branch. I think we are good.

Comment (ShivangiReja) [reply releases/2020-06/dotnet.md:46]: Just FYI, we can use either tag or the master branch with particular release date.

**Pull request merged**